config.yml

```diff
@@ -2,8 +2,10 @@ experiment_name: my_example
 n_repeats: 5
 scenario_params_list:
  - dataset_name:
-     - 'mnist'
-     - 'cifar10'
+     - 'mnist':
+      -
+     - 'cifar10':
+      -
```

Comment:
What was the reasons for making the `init_from_model` parameter a sub-parameter of the datasets? Why not a global parameter?
Oh, I might see: because you wanted the user to be able to activate it only on some datasets? I think this can make sense, but it would make it simpler if we considered that for doing that, the user just have to list another set of scenarios below with its other dataset. No?

Reply:
I make it a sub-parameter as the models for different datasets are not built with the same architecture. If you give a path for one of the model used on a dataset, it won't work on another one. 
Obviously I can do it as list of different scenario to make it clearer. But in that case, we lose part of the 'factorization' thing to write scenarios easier and quicker. I guess it is worth thinking about.

Reply:
In fact, finally it could be interesting to have path to previously learned models regarding architecture and not databases. For the moment we only use one model per dataset but in the future, we could use several models for one datasets. I'll include that in this work. What do you think ?